src/resolve/resolved-manager.c

```diff
@@ -144,6 +148,7 @@ static int manager_process_address(sd_netlink *rtnl, sd_netlink_message *mm, voi
                 break;
 
         case AF_INET6:
+                broadcast.in6 = (struct in6_addr) {0};
```

Comment:
we rely on the GNU C extension that writing just `{}` instead of `{0}` is ok. please use that here too

Reply:
i think it would be easier to just move this up where you declare the variable. given that you need the zero initialization in all codepaths but one, this sounds like it should just be the default value, and only if we the IFA_BROADCAST thing is set we override it.

Reply:
I changed it to use an empty initializer at the declaration above. But now I'm actually thinking that is not correct since apparently only the first (and in this case smallest) union member is initialized. Technically only the v4 addr is ever read though so maybe it's fine?

Reply:
{} will initialize all fields to zero. Unspecified fields are always zero initialized when compound init is used